### PR TITLE
Autonomous

### PR DESCRIPTION
**Pull request by greenblitz4590**
still needs testing for the auto selector

### CODE REVIEW

**Comment by roygold8** [comment]
Let me know when tests are done - CR only after tests!

File: src/main/java/edu/greenblitz/tobyDetermined/Robot.java [hunk 1]
```diff
@@ -78,10 +83,10 @@ public void teleopPeriodic() {
 
     /*
         TODO: Dear @Orel & @Tal, please for the love of god, use the very useful function: schedule(), this will help the code to actually work
-    */
+   :: */
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/tobyDetermined/Robot.java:86]: what

File: src/main/java/edu/greenblitz/tobyDetermined/OI.java [hunk 2]
```diff
@@ -47,15 +48,12 @@ public static boolean isIsHandled() {
...
-
-	public double countB = 0;
-
-
 	private void initButtons() {
-		SwerveChassis.getInstance().setDefaultCommand(new CombineJoystickMovement(true));
-		mainJoystick.Y.onTrue(new InstantCommand(() -> SwerveChassis.getInstance().resetChassisPose()));
-		mainJoystick.POV_UP.onTrue(new InstantCommand(() -> SwerveChassis.getInstance().resetEncodersByCalibrationRod()));
-		mainJoystick.POV_DOWN.onTrue(new ToggleBrakeCoast());
+		mainJoystick.X.onTrue( new SequentialCommandGroup(
```
Comment (roygold8) [inline src/main/java/edu/greenblitz/tobyDetermined/OI.java:52]: Dont merge this one

File: src/main/java/edu/greenblitz/tobyDetermined/Robot.java [hunk 3]
```diff
@@ -79,7 +86,7 @@ public void disabledInit() {
 
     public void teleopInit() {
         CommandScheduler.getInstance().cancelAll();
-        SwerveChassis.getInstance().setIdleModeBrake();
+//        SwerveChassis.getInstance().setIdleModeBrake();
```
Comment (roygold8) [inline src/main/java/edu/greenblitz/tobyDetermined/Robot.java:89]: Why is this commented out?

**Pull request merged**